account_credit_control/views/account_invoice.xml

```diff
@@ -17,7 +17,7 @@
                 <page string="Credit Control"
                       groups="account_credit_control.group_account_credit_control_manager,account_credit_control.group_account_credit_control_user,account_credit_control.group_account_credit_control_info">
                     <group>
-                        <field name="credit_policy_id" widget="selection"
+                        <field name="credit_policy_id"
```

Comment:
I think the use of the widget was intentional to forbid creation of policy from this view